.ci/validation.sh

```diff
@@ -559,10 +560,11 @@ sonarqube)
       SONAR_PR_VARIABLES="-Dsonar.pullrequest.key=$PR"
       SONAR_PR_VARIABLES+=" -Dsonar.pullrequest.branch=$WERCKER_GIT_BRANCH"
       SONAR_PR_VARIABLES+=" -Dsonar.pullrequest.base=master"
-      echo "SONAR_PR_VARIABLES: "$SONAR_PR_VARIABLES
+      echo "SONAR_PR_VARIABLES: ""$SONAR_PR_VARIABLES"
   fi
   if [[ -z $SONAR_TOKEN ]]; then echo "SONAR_TOKEN is not set"; sleep 5s; exit 1; fi
   export MAVEN_OPTS='-Xmx2000m'
+  # shellcheck disable=SC2086
```

Comment:
Please add `until https://github.com/checkstyle/checkstyle/issues/11637` comment above this line, to make sure we come back to this.

Reply:
Changes Done.

Reply:
@nick-mancuso But what I believe is that we have to disable the code here permanently as the shell command will not execute because it asks for a maven goal but by adding " " (SC2086). It sees the mvn goal as null.

Reply:
Please read carefully https://github.com/checkstyle/checkstyle/pull/11650#issuecomment-1130986910

Reply:
Actually, let’s try one more thing to make shellcheck happy before we move on. Instead of “$var”, try “${var}” to use grouping since we have multiple parameters here.

Reply:
I have tried it. Shellcheck doesn't give any error after changing. Let's see if the sonarqube is green or red.

Reply:
Ok, let’s go back to disabled with “until” comment, thanks for trying.

Reply:
Changes Done

Reply:
Resolved.